frontend/src/api/open-hands.ts

```diff
@@ -183,18 +196,63 @@ class OpenHands {
    * Get the VSCode URL
    * @returns VSCode URL
    */
-  static async getVSCodeUrl(): Promise<GetVSCodeUrlResponse> {
-    const { data } =
-      await openHands.get<GetVSCodeUrlResponse>("/api/vscode-url");
+  static async getVSCodeUrl(convoId: string): Promise<GetVSCodeUrlResponse> {
```

Comment:
Nit: Maybe use consistent nomenclature for terms - either `conversationId` or `convoId` in all places

Reply:
Done!